Dockerfile

```
@@ -43,4 +34,13 @@ RUN poetry install
 COPY ./tests ./tests
 
 # Define entrypoint 
-ENTRYPOINT ["poetry", "run", "pytest", "-v"]
\ No newline at end of file
+ENTRYPOINT ["poetry", "run", "pytest", "-v"]
+
+#--- PRODUCTION SPECIFIC STEPS ----
+FROM base as production
+
+# poetry setup
+RUN poetry config virtualenvs.create false --local && poetry install
+
+# start app run command
+CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```

Note that as per Heroku's requirements you now _require_ the user to pass in a PORT environment variable, but your Readme instructions still assume it's set by default. One nice solution is to set a default variable that Heroku can override, so both approaches can be valid:
```suggestion
ENV PORT=5000
CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```